### PR TITLE
Add Azure NIC requirement and clarify azure.json requirement

### PR DESCRIPTION
**Pull request by ivansharamok**
### Proposed changes

Requirements list is missing instruction to enable `IP Forwarding` for each UCP node NIC.

Clarified that `azure.json` is needed only on nodes that run `kubelet` component. At the moment it's not used on Windows workers.

### CODE REVIEW

**Comment by ivansharamok** [comment]
@ollypom @adamancini could you please validate and confirm the changes.

File: ee/ucp/admin/install/cloudproviders/install-on-azure.md [hunk 1]
```diff
@@ -69,7 +70,7 @@ objects are being deployed.
 
 ### Azure Configuration File
 
-For Docker UCP to integrate with Microsoft Azure,each UCP node in your cluster
+For Docker UCP to integrate with Microsoft Azure, each UCP node in your cluster that runs `kubelet` component
```
Comment (ollypom) [inline ee/ucp/admin/install/cloudproviders/install-on-azure.md]: My nit `each Linux UCP node in your cluster`. People may think a worker node is a swarm worker, therefore no kubelet component will be deployed :)

File: ee/ucp/admin/install/cloudproviders/install-on-azure.md [hunk 2]
```diff
@@ -45,6 +45,7 @@ to successfully deploy Docker UCP on Azure:
   Configuration](#considerations-for-ipam-configuration).
 - All UCP worker and manager nodes need to be attached to the same Azure
   Subnet.
+- All UCP Nodes NIC's (Network Interface) must have IP forwarding enabled.
```
Comment (ollypom) [inline ee/ucp/admin/install/cloudproviders/install-on-azure.md]: Do we have a link to how to enable this in Azure?

Comment (ivansharamok) [reply ee/ucp/admin/install/cloudproviders/install-on-azure.md]: Looks like this is not a requirement at all.

**Comment by ivansharamok** [comment]
Closing this PR as `UCP Nodes NIC` change is not necessary and `azure.json` change needs further clarification.
Will make changes and submit another PR.

File: ee/ucp/admin/install/cloudproviders/install-on-azure.md (hunk 1 above)
Comment (ivansharamok) [reply ee/ucp/admin/install/cloudproviders/install-on-azure.md]: will adopt proposed change and submit a new PR on this.

Comment (ivansharamok) [reply ee/ucp/admin/install/cloudproviders/install-on-azure.md]: @olly, how about this:
```
each Linux UCP node in your cluster that runs `kubelet` component needs an Azure configuration file, `azure.json`. Note, that Swarm only nodes either Linux or Windows do not run `kubelet` component and therefore do not require `azure.json` file.
```

**Pull request closed without merging**